aws/ci_build

```diff
@@ -45,7 +45,7 @@ def upload_log_and_get_link(key, body, metadata)
         body,
         {metadata: metadata}
       )
-  " <a href='#{log_url}'>☁ Log on S3</a>"
+  " <a href='#{log_url}'>  Log on S3</a>"
```

Comment:
Is it better to use the unicode escape or the HTML escape? Note that I chose the former, though I'm happy to do the latter if more appropriate.

And, out of playful curiosity, why do we mind the cloud disappearing?